experimental/examples/wheel/BUILD

```diff
@@ -131,6 +145,22 @@ py_wheel(
     ],
 )
 
+py_wheel(
+    name = "use_genrule_with_dir_in_outs",
+    distribution = "use_genrule_with_dir_in_outs",
+    python_tag = "py3",
+    version = "0.0.1",
+    deps = [
+        ":example_pkg_with_data"
+    ]
+)
+
+genrule(
```

Comment:
nit: could this be declared closer to where it gets referred (on line 34)?

Reply:
Updated